### PR TITLE
Multi-language documentation for Injecting Kafka Producer Beans example

### PR DESCRIPTION
**Pull request by wetted**

### CODE REVIEW

**Comment by wetted** [comment]
@timyates thanks for helping

File: test-suite-groovy/src/test/groovy/io/micronaut/kafka/docs/producer/inject/Book.groovy [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package io.micronaut.kafka.docs.producer.inject
+
+import io.micronaut.serde.annotation.Serdeable
+
+@Serdeable
+class Book {
+    String title
+
+    Book(String title) {
```
Comment (sdelamo) [inline test-suite-groovy/src/test/groovy/io/micronaut/kafka/docs/producer/inject/Book.groovy:9]: remove constructor. Annotate `Book` with `@Canonical` or use `new Book(title: "foo")`

**Review by sdelamo — APPROVED** [review]

File: test-suite/src/test/java/io/micronaut/kafka/docs/producer/inject/BookSenderTest.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    // tag::test[]
+    @Test
+    void testBookSender() {
+
+        ApplicationContext ctx = ApplicationContext.run( // <1>
+            Map.of("kafka.enabled", "true","spec.name", "BookSenderTest")
+        );
+        BookSender bookSender = ctx.getBean(BookSender.class); // <2>
+        Book book = new Book("The Stand");
+        bookSender.send("Stephen King", book);
```
Comment (timyates) [inline test-suite/src/test/java/io/micronaut/kafka/docs/producer/inject/BookSenderTest.java:19]:
```suggestion

        try (ApplicationContext ctx = ApplicationContext.run( // <1>
            Map.of("kafka.enabled", "true", "spec.name", "BookSenderTest")
        )) {
            BookSender bookSender = ctx.getBean(BookSender.class); // <2>
            Book book = new Book("The Stand");
            bookSender.send("Stephen King", book);
        }
```

Comment (timyates) [reply test-suite/src/test/java/io/micronaut/kafka/docs/producer/inject/BookSenderTest.java:19]: Sonar also points out that this doesn't have any assertions

**Review by timyates — CHANGES_REQUESTED** [review]
The tests should assert something 😎

**Review by timyates — APPROVED** [review]

**Pull request merged**